Fix for bucket event limits (#1337)
minio/docs: source/administration/monitoring/bucket-notifications.rst

.. note::

   For synchronous and asynchronous events, MinIO maintains a per-remote queue where it stores unsent and pending events.
   The queue limit defaults to ``100000``.

>    MinIO discards new events if the queue fills with unsent messages.

Reviewer: ```suggestion
   MinIO discards new events when the queue is full.
```

Alternatively, the queue can have both unsent and pending events, but do we only discard if the queue is full of unsents with no pendings? My guess is once we hit 10K, we discard new events until the queue is 9,999 or less.
